Nola-Devs/Nola-Devs-v2
# Redesign Home Page UI Based on Updated Figma Specifications

# 🚀 Description

This PR introduces a redesign of the home page UI to align with the latest Figma designs. 

## 💻 Type of change


## 📝 How has this been tested and how can we Reproduce?

- [X] Visual inspection against Figma designs to ensure accuracy in implementation
- [X] Compatibility testing across different browsers and devices to ensure consistent user experience.
- [X] Functionality testing of new features added to the home page.

Instructions for reproduction:

1. Navigate to the homepage.
2. Verify new UI elements against the provided Figma design.
3. Interact with new features and note any discrepancies or issues.

## 📸 Any screenshots or links to points in your code or references elsewhere as needed

![image](https://github.com/Nola-Devs/Nola-Devs-v2/assets/66075680/0910bddd-5233-4443-8d26-a02b08a035f4)
![image](https://github.com/Nola-Devs/Nola-Devs-v2/assets/66075680/b198fdc4-9f10-47b7-a927-0c2a9663fc1c)


## ✅ Checklist:

- [X] I have included in my PR description where corresponding changes to the documentation are needed
